llvm/lib/Target/RISCV/RISCVInstrInfoCOREV.td

```diff
@@ -882,7 +882,6 @@ let Predicates = [HasExtXcvalu] in {
   def : Pat<(srl (add (sub GPR:$rd, GPR:$rs1), (roundBit GPR:$rs2)), GPR:$rs2),
             (CV_SUBURNR GPR:$rd, GPR:$rs1, GPR:$rs2)>;
 
-  def : PatCoreVAluGpr<"abs", "ABS">;
   def : PatCoreVAluGprGpr<"slet", "SLET">;
   def : PatCoreVAluGprGpr<"sletu", "SLETU">;
```

Comment:
Will need to change SLET to SLE and SLETU to SLEU when rebasing to solve the merge conflict